pkg/apis/config/features_test.go

```diff
@@ -114,6 +116,24 @@ func TestFeaturesConfiguration(t *testing.T) {
 		data: map[string]string{
 			"multi-container": "Disabled",
 		},
+	}, {
+		name:    "multi-container-probing Allowed",
+		wantErr: false,
+		wantFeatures: defaultWith(&Features{
+			MultiContainerProbing: Allowed,
+		}),
+		data: map[string]string{
+			"multi-container-probing": "Allowed",
+		},
+	}, {
+		name:    "multi-container-probing Disabled",
+		wantErr: false,
+		wantFeatures: defaultWith(&Features{
+			MultiContainerProbing: Disabled,
+		}),
+		data: map[string]string{
+			"multi-container-probing": "Disabled",
+		},
```

Comment:
For consistency reasons I think you need another test with name: "multi-container-probing Enabled".

Reply:
I think we have it [here](https://github.com/knative/serving/pull/14853/files/7b301934c393f8afb83201d56ea498804230beef#diff-0480360542ba562777855d37f59cd0a5ab56bac6af41b1424660b3b80481890cR63)

Reply:
I copied the "multi-container" one. It has only two cases. I think because it already tests Enabled in the `features Enabled` test. But I'm fine with adding that as well.